### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by aaroncolichia**
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [ ] All tests passed: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript: 30 points

**Checkpoint 2**
- [ ] The application works as it should: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript and the DOM API: 30 points

**Checkpoint 3**
- [ ] Use of React: 25 points
- [ ] Accesses an API: 25 points
- [ ] Proper use of documentation (commenting on code): 25 points
- [ ] The application functions as it should: 25 points

### CODE REVIEW

File: 03week/ticTacToe.js [hunk 1]
```diff
@@ -25,22 +25,56 @@ function printBoard() {
...
+// two players will select pairs of integers corresponding to coordinates on an
+//  X-Y axis
+// program will store each players selection
+// a player cannot select an already played coordinate
+// each turn the program will look for all possible win scenarios
+// if a win is determined the winner is announced and board is cleared
+
 function ticTacToe(row, column) {
   // Your code here
+  if (row < 3 && column < 3 && !(board[row][column]).trim()) {
```
Comment (ericbarrs) [inline 03week/ticTacToe.js]: Excellent use of of your if statement. I was able to break your code by putting in a negative number.

File: 03week/ticTacToe.js [hunk 2]
```diff
@@ -25,22 +25,56 @@ function printBoard() {
...
+    board[row][column] = playerTurn
+    if (horizontalWin() || verticalWin() || diagonalWin()) {
+      console.log("Player " + playerTurn + " is the winner!.");
```
Comment (ericbarrs) [inline 03week/ticTacToe.js]: you game does not show the final result after the game is won. Also if there is a tie the game does not restart.

File: 03week/ticTacToe.js (hunk 1 above)
Comment (aaroncolichia) [reply 03week/ticTacToe.js]: done

File: 03week/ticTacToe.js (hunk 2 above)
Comment (aaroncolichia) [reply 03week/ticTacToe.js]: done

**Pull request merged**